packages/deployment/upgrade-test/Dockerfile

```
@@ -1,27 +1,20 @@
 # Defaults
-ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals@sha256:ac10c09b5927d759d37b1525b5a0bc4aeb3034df8d0f0fbb15b51203555ffb1b
+ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals:pr33
```

https://github.com/Agoric/agoric-3-proposals/pull/33 merged. I guess this isn't `main` because that flow is failing?

Please leave a comment here about why it isn't `main` as it should be. We should unbreak that. 
```suggestion
// Once https://github.com/Agoric/agoric-3-proposals/pull/37 merges, s/pr33/main
ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals:pr33
```

---

Even if it was pushed to the `main` tag, I do not feel confident using `main` until https://github.com/Agoric/agoric-3-proposals/issues/25 is resolved.

---

I think your concern is about this CI starting to fail once `main` includes the upgrade being tested here.

So even after 25, you wouldn't want `:main`, right? You'd want `:upgrade12` or something like that. 

Given that I propose a different comment.
```suggestion
# Don't use `:main` because once this upgrade is incorporated into it, this test would break
ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals:pr33
```